2wp-app simplify the documentation (#929)
rsksmart/devportal: content/rsk-devportal/guides/two-way-peg-app/glossary.md

**2 way peg app:**

> The [2 way peg app](https://2wp-app.rootstock.io/) is a frontend application developed in Vue.js and typescript. The or 2 way peg app is a web application that fosters the interaction between the bitcoin blockchain and the RSK network for faster exchange of BTC and RBTC. See the [github repo](https://github.com/rsksmart/2wp-app).

Reviewer: ```suggestion
The [2 way peg app](https://2wp-app.rootstock.io/) is a frontend application developed in Vue.js and typescript. The 2 way peg app is a web application that fosters the interaction between the bitcoin blockchain and the Rootstock network for simpler exchange of BTC and RBTC. See the [github repo](https://github.com/rsksmart/2wp-app).
```

A software wallet is an application that is installed on a computer or smartphone. The private keys are stored on the computer or smartphone.

**Testnet:**

> - This is a testing network used for testing and development purposes, assets on the Testnet have zero value, funds used in this network are called Test tokens (tRBTC), they can be gotten through a faucet that dispenses tokens. These tokens are utility tokens that are required to operate certain DApps. Developers of those DApps also need to test them on the Testnet, and hence these are provided as a convenience for them. The RSK network provides a cryptocurrency faucet. The tRBTC faucet provides the cryptocurrency required to pay for gas fees on the RSK Testnet. See how to get tRBTC using the [RSK Faucet](https://faucet.rootstock.io/).

Reviewer: ```suggestion
- This is a testing network used for testing and development purposes, assets on the Testnet have zero value, funds used in this network are called Test tokens (tRBTC), they can be gotten through a faucet that dispenses tokens. These tokens are utility tokens that are required to operate certain DApps. Developers of those DApps also need to test them on the Testnet, and hence these are provided as a convenience for them. The Rootstock network provides a cryptocurrency faucet. The tRBTC faucet provides the cryptocurrency required to pay for gas fees on the Rootstock Testnet. See how to get tRBTC using the [Rootstock Faucet](https://faucet.rootstock.io/).
```

## Resources
- 2 way peg app frontend [repo](https://github.com/rsksmart/2wp-app)
- 2 way peg app backend [repo](https://github.com/rsksmart/2wp-api)

> - How to get [RBTC using RSK’s built in Powpeg](https://developers.rootstock.io/guides/get-crypto-on-rsk/powpeg-btc-rbtc/)

Reviewer: ```suggestion
- How to get [RBTC using Rootstock's built in Powpeg](https://developers.rootstock.io/guides/get-crypto-on-rsk/powpeg-btc-rbtc/)
```

> - [RSK Testnet Faucet](https://faucet.rootstock.io/)

Reviewer: ```suggestion
- [Rootstock Testnet Faucet](https://faucet.rootstock.io/)
```
